libraries/browser-tracker-core/src/tracker/types.ts

```diff
@@ -189,15 +199,15 @@ export type TrackerConfiguration = {
    */
   connectionTimeout?: number;
   /**
-   * Condifugration for Anonymous Tracking
```

Comment:
Some of the typos are quite funny :D

Reply:
Have you never tried to Condifugrate your trackers before?